com.unity.multiplayer.mlapi/Prototyping/NetworkTransform.cs

```diff
@@ -297,7 +266,7 @@ private void FixedUpdate()
                 return;
             }
 
-            if (CanUpdateTransform())
+            if (this.HasAuthority(m_TransformAuthority))
```

Comment:
HasAuthority feels like it should be part of the authority utility. The utility could actually be a component contained by NetworkTransform?